ros2doctor/ros2doctor/command/doctor.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from ros2cli.command import add_subparsers
+from ros2cli.command import CommandExtension
+from ros2cli.verb import get_verb_extensions
+
+
+class DoctorCommand(CommandExtension):
+    """Various doctor related sub-commands."""
+
+    def add_arguments(self, parser, cli_name):
+        self._subparser = parser
+        verb_extensions = get_verb_extensions('ros2doctor.verb')
+        add_subparsers(
+            parser, cli_name, '_verb', verb_extensions, required=False)
+
+    def main(self, *, parser, args):
+        if not hasattr(args, '_verb'):
+            # in case no verb was passed
+            self._subparser.print_help()
+            return 0
```

Comment:
`roswtf` when invoked on its own would run all checks. I recommend keeping the same behavior with `ros2 doctor` running every check by default. If someone calls `roswtf`, it's because they don't know what the problem is, and the the tool tells them what looks broken.

Reply:
I'm actually not sure how to add alias to a command. In this case I only added `wtf` to `entry_points` in `setup.py`, and that resulted in behaviours I didn't expect. But yes, `ros2 wtf` is supposed to behave the same way as `ros2 doctor`. Do you know a cli command that also has alias set up? Maybe I could use that repo as reference.

Reply:
I was referring to the ros1 tool `roswtf`; I wasn't suggesting adding an alias. Rephrasing:

Does `ros2 doctor` run all checks when called without any arguments, just like the ros 1 tool `roswtf` does?